integrated_channels/integrated_channel/transmitters/content_metadata.py

```diff
@@ -32,49 +32,48 @@ def __init__(self, enterprise_configuration, client=IntegratedChannelApiClient):
             client=client
         )
 
-    def _log_info(self, msg, content_id=None):
+    def _log_info(self, msg, course_or_course_run_key=None):
         LOGGER.info(
             generate_formatted_log(
                 self.enterprise_configuration.channel_code(),
                 self.enterprise_configuration.enterprise_customer.uuid,
                 None,
-                content_id,
+                course_or_course_run_key,
                 msg
             )
         )
 
-    def _log_error(self, msg):
+    def _log_error(self, msg, course_or_course_run_key=None):
         LOGGER.error(
             generate_formatted_log(
                 self.enterprise_configuration.channel_code(),
                 self.enterprise_configuration.enterprise_customer.uuid,
                 None,
-                None,
+                course_or_course_run_key,
                 msg
             )
         )
 
-    def transmit(self, create_payload, update_payload, delete_payload, content_updated_mapping, **kwargs):
+    def _log_info_for_each_item_map(self, item_map, msg):
```

Comment:
helper to DRY up logging a message-per-item in a transmission item map, which is done in a few places.